workflow-service/src/test/java/com/redhat/parodos/user/service/UserServiceImplTest.java

```diff
@@ -0,0 +1,120 @@
+package com.redhat.parodos.user.service;
+
+import com.redhat.parodos.user.dto.UserResponseDTO;
+import com.redhat.parodos.user.entity.User;
+import com.redhat.parodos.user.repository.UserRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.modelmapper.ModelMapper;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static java.lang.String.format;
+import static org.junit.jupiter.api.Assertions.*;
+
+class UserServiceImplTest {
+
+	private UserRepository userRepository;
+
+	private UserServiceImpl service;
+
+	@BeforeEach
+	public void initEach() {
+		this.userRepository = Mockito.mock(UserRepository.class);
+		this.service = new UserServiceImpl(this.userRepository, new ModelMapper());
+	}
+
+	@Test
+	void saveTestWithValidData() {
+		// given
+		User user = getSampleUser("test");
+		Mockito.when(this.userRepository.save(Mockito.any(User.class))).thenReturn(user);
+
+		// when
+		UserResponseDTO res = this.service.save(user);
+
+		// then
+		assertNotNull(res);
+		assertEquals(res.getId().toString(), user.getId().toString());
```

Comment:
`res.getId()` returns `String` so you can simplify this assertion as
`assertEquals(res.getId(), user.getId().toString());`